src/vivarium_census_prl_synth_pop/tools/jobmon.py

```diff
@@ -0,0 +1,86 @@
+import shutil
+import uuid
+from pathlib import Path
+from typing import Union
+
+from jobmon.client.tool import Tool
+from loguru import logger
+
+
+def run_make_results_workflow(
+    raw_output_dir: Path,
+    final_output_dir: Path,
+    mark_best: bool,
+    test_run: bool,
+    artifact_path: Union[str, Path],
+    queue: str,
+    peak_memory: int,
+    max_runtime: str,
+    project: str,
+) -> None:
+    """Creates and runs a jobmon workflow to build results datasets
+    from the raw data output by observers
+    """
+    logger.info(
+        "Starting jobmon 'make_results' workflow. Results will be written to "
+        f"{final_output_dir}"
+    )
+    wf_uuid = uuid.uuid4()
+
+    # Deal with boolean args - click either the flag or nothing, not True/False
+    # TODO: there's got to be a better way to do this?
+    mark_best_arg = "--mark-best" if mark_best else ""
+    test_run_arg = "--test-run" if test_run else ""
+
+    # Create tool
+    tool = Tool(name="vivarium_census_prl_synth_pop.make_results")
+
+    # Create a workflow
+    workflow = tool.create_workflow(name=f"make_results_workflow_{wf_uuid}")
+
+    # Create task templates
+    template_make_results = tool.get_task_template(
+        template_name="make_results_template",
+        default_compute_resources={
+            "queue": queue,
+            "cores": 1,
+            "memory": peak_memory,
+            "runtime": max_runtime,
+            "project": project,
+            "stdout": str(final_output_dir),
+            "stderr": str(final_output_dir),
+        },
+        default_cluster_name="slurm",
+        command_template=(
+            f"{shutil.which('jobmon_make_results_runner')} "
+            "{raw_output_dir} "
+            "{final_output_dir} "
+            "{mark_best} "
```

Comment:
It occurs that we may need a slightly more complicated workflow than this. We will want to wait until all jobs have completed successfully before we mark the results as best or latest. This can be addressed in a later PR though.